patricktunez123/Andela_Teamwork
# #168837585 Remmove dead links and make ui look more beautiful

#### What does this PR do?
Remove dead links within the entire app
Add edit and flag icons
Change the size of  image on article page
#### Description of Task to be completed?
Got dead links removed within the entire app
Have edit and flag icons changed
Have the size of  image on article page changed
#### How should this be manually tested?
After cloning this repo, cd into it and type npm install, than npm run, and then in your browser type localhost:3000/UI, you should see all of these changes above
#### What are the relevant pivotal tracker stories?
#168837585
#### Screenshots
![1](https://user-images.githubusercontent.com/53449099/66182323-e8552600-e674-11e9-8cae-5359ecdcb5dc.PNG)

![2](https://user-images.githubusercontent.com/53449099/66182332-f014ca80-e674-11e9-949e-4734641493f6.PNG)

![3](https://user-images.githubusercontent.com/53449099/66182338-f4d97e80-e674-11e9-9162-caf3c2f0fe0c.PNG)

![4](https://user-images.githubusercontent.com/53449099/66182344-fc992300-e674-11e9-82d0-1b92f8f8701a.PNG)
